xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
How does it not break api? It currently sets the writer info. Removing it means plugins using it will now not set writer info. And api isn't broken for krypton. Not sure why you think that...